includes/class-amp-story-post-type.php

```diff
@@ -222,6 +240,7 @@ public static function enqueue_block_editor_assets() {
 			return;
 		}
 
+		// @todo Move the following styles to register_styles method, and scripts to a new register_scripts method which runs at wp_default_scripts?
```

Comment:
What I meant here was that potentially there could be methods that run at `wp_default_scripts` and `wp_default_styles` which then do `$scripts->add()` and `$styles->add()` respectively. That would then leave the existing `enqueue_block_editor_assets` method to just `wp_enqueue_script($handle)` and `wp_enqueue_style($handle)`.

But this is not really necessary for any scripts and styles that are enqueued in multiple places.

Reply:
Ah, that makes sense. Sorry for missing that.

It looks like you renamed some methods in https://github.com/ampproject/amp-wp/pull/1944/commits/80b3a7ea6558ec9b4c8a7bf71267846110a0d593. Let me know if you'd like me to apply your suggestion of calling `$scripts->add()` and `$styles->add` at `wp_default_scripts` and `wp_default_styles`